test/integration/mon/serverspec/mon_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'serverspec'
+
+set :backend, :exec
+
+load_thres = if %w(ppc64 ppc64le).include?(os[:arch])
+               '-w 14,9,4 -c 18,13,8'
```

Comment:
Correct, these values are set based on the test kitchen environment. The ``total_cpu`` value for the nodes are 20, so using that you can see that the values should become ``-w 90,85,80 -c 170,165,160`` on the openpower nodes.

Is that acceptable?